ex/ThirdPartyDemos/martin-doyle/02 - Http Client Server ORM/source/main.pas

```diff
@@ -0,0 +1,102 @@
+unit main;
+
+interface
+
+uses
+  {$I mormot.uses.inc}
+  {$ifdef MSWINDOWS}
+  Windows,
+  {$endif MSWINDOWS}
+  Messages, SysUtils, Variants, Classes, Graphics, Controls, Forms,
```

Comment:
If I understand correctly, the samples are for Delphi VCL only yet.

Reply:
Tested with Lazarus on Windows and Debian Linux as well. This is what I like very much about mORMot. It works even on my ARM64 server.